engine/apps/public_api/tests/test_on_call_shifts.py

```diff
@@ -165,6 +165,53 @@ def test_get_override_on_call_shift(make_organization_and_user_with_token, make_
     assert response.data == result
 
 
+@pytest.mark.django_db
+def test_get_web_override_shift(
+    make_organization_and_user_with_token, make_user_for_organization, make_on_call_shift, make_schedule
+):
+    organization, user, token = make_organization_and_user_with_token()
+    other_user = make_user_for_organization(organization)
+    client = APIClient()
+
+    start_data = timezone.now().replace(microsecond=0)
+    schedule = make_schedule(organization, schedule_class=OnCallScheduleCalendar)
+    data = {
+        "name": "override shift",
+        "start": start_data,
+        "rotation_start": start_data,
+        "duration": timezone.timedelta(hours=9),
+        "schedule": schedule,
+    }
+    on_call_shift = make_on_call_shift(
+        organization=organization,
+        source=CustomOnCallShift.SOURCE_WEB,
+        shift_type=CustomOnCallShift.TYPE_OVERRIDE,
+        **data,
+    )
+    on_call_shift.add_rolling_users([[other_user]])
+    on_call_shift.users.add(user)
```

Comment:
When creating override shifts, the `users` field should be empty. We only expect `rolling_users` to be set up in this case.